Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -219,25 +608,29 @@ The following rules need to be followed, in addition to the guidelines described
 
     Buildings.Controls.OBC.CDL.Interfaces.RealInput TZon(
       final unit="K",
-      displayUnit="degC") "Measured zone air temperature";
+      final displayUnit="degC") "Measured zone air temperature";
```

Comment:
Display unit must not be final. Users may want to change this to IP.

Reply:
That makes perfect sense - however we've been using final with the displayUnit throughout the OBC package. It should be easy to replace.

Reply:
#2045